src/sentry/incidents/subscription_processor.py

```diff
@@ -142,13 +142,15 @@ def process_update(self, subscription_update):
             if alert_operator(
                 aggregation_value, trigger.alert_threshold
             ) and not self.check_trigger_status(trigger, TriggerStatus.ACTIVE):
+                metrics.incr("incidents.alert_rules.hit_trigger_threshold")
                 with transaction.atomic():
                     self.trigger_alert_threshold(trigger)
             elif (
                 trigger.resolve_threshold is not None
                 and resolve_operator(aggregation_value, trigger.resolve_threshold)
                 and self.check_trigger_status(trigger, TriggerStatus.ACTIVE)
             ):
+                metrics.incr("incidents.alert_rules.hit_resolve_threshold")
```

Comment:
Would this and the trigger metric be better as tags? If the metric name was `incidents.alert_rules.threshold` with hit/resolved as tags you could graph the two states in datadog a bit more easily.

Reply:
Ahh I see. I'm still used to the old graphite way of doing things, this makes sense though.